debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/SourceInfo.java

```diff
@@ -81,6 +81,7 @@
     public static final String XMIN_KEY = "xmin";
     public static final String LSN_KEY = "lsn";
     public static final String LAST_SNAPSHOT_RECORD_KEY = "last_snapshot_record";
+    public static final String ORIGIN_NAME_KEY = "origin_name";
```

Comment:
I think just "origin" would suffice for the key.

Reply:
Hi,
1.  According to my tests, there is no origin value by default.  when we use
`select * from pg_replication_ORIGIN_session_setup ('origin_name')` later,
All subsequent operations on this connection will have the origin value attached. After `BEGIN`, each transaction.

2. I will do some test , but  the test of this project is too complicated. I need some time to study it.
3. I will update the docs.

Thanks.